tests/stress_benchmark_test.ts

```diff
@@ -0,0 +1,287 @@
+              if ("mantaPay.ToPrivate" == event) {
+                allSuccesses++;
+              }
+            });
+          }
+        });
+      await api.tx.mantaPay
+        .toPrivate(
+          reclaimsBuffer.subarray(
+            reclaimsStart + test_config.mint_size,
+            reclaimsStart + 2 * test_config.mint_size
+          )
+        )
+        .signAndSend(sender, { nonce: -1 }, ({ events = [], status }) => {
+          if (status.isInBlock) {
+            console.log("Included at block hash", status.asInBlock.toHex());
+            console.log("Events:");
+            events.forEach(({ event: { data, method, section }, phase }) => {
+              let event = section + "." + method;
+              console.log("event: ", event);
+              if ("mantaPay.ToPrivate" == event) {
+                allSuccesses++;
+              }
+            });
+          }
+        });
+      if (i == test_config.start_iteration + test_config.tests_iterations - 1) {
+        const unsub = await api.tx.mantaPay
+          .toPublic(
+            reclaimsBuffer.subarray(
+              reclaimsStart + 2 * test_config.mint_size,
+              reclaimsStart +
+                2 * test_config.mint_size +
+                test_config.reclaim_size
+            )
+          )
+          .signAndSend(sender, { nonce: -1 }, ({ events = [], status }) => {
+            if (status.isInBlock) {
+              console.log("Included at block hash", status.asInBlock.toHex());
+              console.log("Events:");
+              events.forEach(({ event: { data, method, section }, phase }) => {
+                let event = section + "." + method;
+                console.log("event: ", event);
+                if ("mantaPay.ToPublic" == event) {
+                  allSuccesses++;
+                }
+              });
+            } else if (status.isFinalized) {
+              lastFinalized = true;
+              let endTime = performance.now();
+              totalTime = endTime - startTime;
+              // Convert to seconds
+              totalTime = totalTime / 1000;
+              unsub();
+            }
+          });
+      } else {
+        await api.tx.mantaPay
+          .toPublic(
+            reclaimsBuffer.subarray(
+              reclaimsStart + 2 * test_config.mint_size,
+              reclaimsStart +
+                2 * test_config.mint_size +
+                test_config.reclaim_size
+            )
+          )
+          .signAndSend(sender, { nonce: -1 }, ({ events = [], status }) => {
+            if (status.isInBlock) {
+              console.log("Included at block hash", status.asInBlock.toHex());
+              console.log("Events:");
+              events.forEach(({ event: { data, method, section }, phase }) => {
+                let event = section + "." + method;
+                console.log("event: ", event);
+                if ("mantaPay.ToPublic" == event) {
+                  allSuccesses++;
```

Comment:
why not unsub here and at all the others except line 203 where youre waiting for finalization?

Reply:
No reason , not even sure what the benefit is so ill just remove it